src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -0,0 +1,428 @@
+                        + $"\n**Action :**  '{updateHistoryError.Action}'"
+                        + $"\n**Last seen :**  '{updateHistoryError.Timestamp}'";
+                    replaceDesc = (argument, issueBody) => true;
+                    break;
+                }
+            }
+            return CreateOrUpdateGithubIssueAsync(updateHistoryError, issueRepo, replaceDesc, description);
+        }
+
+        /*      Eg: Body of Comment :
+                The following errors have been detected when attempting to update dependencies in 'https://github.com/maestro-auth-test/maestro-test2' 
+
+                [marker]: <> (subscriptionId: 'a9f0536e-15cc-4d1f-79a4-08d79acf8bba', method: 'UpdateAssetsAsync', errorMessage: 'build: 14	Unexpected error processing action: Validation Failed')
+                **SubscriptionId:** 'a9f0536e-15cc-4d1f-79a4-08d79acf8bba'
+                **Source Repository :**  'https://github.com/maestro-auth-test/maestro-test1'
+                **Target Repository :**  'https://github.com/maestro-auth-test/maestro-test2'
+                **Branch Name :**  '38'
+                **Error Message :**  'build: 14	Unexpected error processing action: Validation Failed'
+                **Method :** 'UpdateAssetsAsync'
+                **Action :** 'Updating assets for subscription: ee8cdcfb-ee51-4bf3-55d3-08d79538f94d'
+                **Last seen :** 'Maestro.Data.RepositoryBranchUpdateHistoryEntry'
+
+                The Regex : (\w+): '([^,\n]+)'(?:, |\)) captures (subscriptionId: 'a9f0536e-15cc-4d1f-79a4-08d79acf8bba', method: 'UpdateAssetsAsync', errorMessage: 'build: 14	Unexpected error processing action: Validation Failed')
+                the above string and puts it into two groups. 
+
+                group[1]            group[2]
+                subscriptionId      a9f0536e-15cc-4d1f-79a4-08d79acf8bba
+                method              UpdateAssetsAsync
+                errorMessage        build: 14	Unexpected error processing action: Validation Failed
+
+                This is put into a dictionary, where group[1] is put in as key and group[2] as value.
+        */
+        private static Dictionary<string, string> ParseIssueCommentBody (string body)
+        {
+            var arguments = new Dictionary<string, string>();
+            var matches = CommentMarker.Matches(body);
+
+            foreach (Match match in matches) {
+                var group = match.Groups;
+                arguments.Add(group[1].Value, group[2].Value);
+            }
+            return arguments;
+        }
+
+        private string ParseUpdateAssetsAsync(RepositoryBranchUpdateHistoryEntry updateHistoryError)
+        {
+            var description = "";
+            string subscriptionId = GetSubscriptionId(updateHistoryError.Arguments);
+            Guid subscriptionGuid = GetSubscriptionGuid(subscriptionId); 
+            var subscription = (from sub in
```

Comment:
Pretty sure I've given feedback to change the syntax for this one.

Reply:
yes you did. I changed the query. If i changed the initialization part, it complains about which one to use octokit or Meastro one. So I used var here to not have any confusion. 
There is one more place I use Repository which is an octokit one.